frontend/public/locales/en/personal-information.json

```
@@ -87,14 +86,53 @@
         "address-change-confirm": "Confirm"
       },
       "change-of-address": "Change of home address",
-      "from": "From:",
-      "to": "To:",
+      "previous-home-address": "Previous home address",
+      "new-home-address": "New home address",
+      "button": {
+        "confirm": "Confirm",
+        "cancel": "Cancel"
+      }
+    }
+  },
+  "mailing-address": {
+    "edit": {
+      "page-title": "Change mailing address",
+      "breadcrumbs": {
+        "home": "Home",
+        "personal-information": "Personal information",
+        "mailing-address-change": "Change mailing address"
+      },
+      "field": {
+        "address": "Adddress",
```

```suggestion
        "address": "Address",
```

This typo also occurs on the Home Address edit page and in `/fr/personal-information.json`

---

fixed